docs/components/tasklist/userguide/using-tasklist.md

```diff
@@ -29,29 +29,33 @@ Change the list of tasks by applying filters. You can also collapse and expand t
 You can choose which tasks you want to see:
 
 - All open
-- Claimed by me
-- Unclaimed
+- Assigned to me
+- Unassigned
 - Completed
 
-Initially, we have no **Claimed by me** tasks.
+You can also order the tasks on the left menu, clicking on the icon ![order-icon](img/order-icon.png):
```

Comment:
Rephrase to

"Click on the icon ![order-icon](img/order-icon.png) to order the tasks. You can order them by the date of creation, the due date, or the follow up date.

The follow up date defines the latest time you should start working on a task, helping you to prioritize work.
The due date provides a deadline when the task should be finished."